deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -230,6 +230,11 @@ def get(self, request, name, stage):
             "placements": placements,
             "cells": cells,
             "arches": arches,
+            "bandwidthWeights": [
+                {"name": "default"},
+                {"name": "vpc-1"},
+                {"name": "ebs-1"},
+            ],
```

Comment:
A constant could be used to avoid repeating the list below

Reply:
I plan to retrieve this list from the Rodimus database after adding the necessary entries to the table.